ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,467 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
+
+import ga4gh.protocol as protocol
+import ga4gh.exceptions as exceptions
+
+import rdflib
+from rdflib import URIRef
+from rdflib import Graph
+
+
+class G2PDataset:
+    """
+    an rdf object store
+    """
+
+    # keep the graph as a static variable
+    g = None
+    # overwrite tests/unit/cgd-test.ttl
+    record_test = False
+    #
+    g2 = Graph()
+
+    def __init__(self, is_prod):
+
+        self.search_query = """
+            PREFIX OBAN: <http://purl.org/oban/>
+            PREFIX OBO: <http://purl.obolibrary.org/obo/>
+            PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+            PREFIX faldo: <http://biohackathon.org/resource/faldo#>
+            PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+            SELECT %PROPERTIES%
+                WHERE {
+                    ?s    a OBAN:association .
+                    ?s  OBAN:association_has_subject ?l .
+                    ?l rdfs:label ?location_label  .
+                    %LOCATION%
+                    ?s  OBO:RO_has_environment  ?drug .
+                    ?drug  rdfs:label ?drug_label  .
+                    ?s  OBAN:association_has_object  ?d .
+                    ?d  rdfs:label ?disease_label  .
+                    ?d rdf:type ?disease .
+                    ?s  OBAN:association_has_object_property  ?evidence .
+                    OPTIONAL {  ?evidence  rdfs:label ?evidence_label } .
+              %FILTER%
+            }
+            """
+
+        if (G2PDataset.g):
```

Comment:
Two issues here. Brackets are unnecessary, and also we follow PEP8's guidance and always compare with  ``None`` explicitly, ie.
```python
if self.rdfGraph is None:
    return
```
However, the intention of this is also confusing. Why do we need a class variable rather than an instance variable? There should only be one instance of this class in use, so what's the point of the class variable? This would be much simplified I think if we just had an instance variable ``self._rdfGraph = rdflib.Graph()``.